myriad-scheduler/src/test/resources/myriad-config-test-default-with-docker-info.yml

```diff
@@ -20,10 +20,10 @@ checkpoint: false
 frameworkFailoverTimeout: 43200000
 frameworkName: MyriadTest
 frameworkRole: test
-frameworkUser: hduser     # User the Node Manager runs as, required if nodeManagerURI set, otherwise defaults to the user
+frameworkUser: hduser     # User the Node Manager runs as, required if nodemanagerURI set, otherwise defaults to the user
                           # running the resource manager.
 frameworkSuperUser: root  # To be depricated, currently permissions need set by a superuser due to Mesos-1790.  Must be
-                          # root or have passwordless sudo. Required if nodeManagerURI set, ignored otherwise.
+                          # root or have passwordless sudo. Required if nodemanagerURI set, ignored otherwise.
```

Comment:
Yeah, agreed, will hold on future changes unless there's a really
compelling reason to do so.

--John

On Wed, Jun 1, 2016 at 1:18 AM, Darin <notifications@github.com> wrote:

> In
> myriad-scheduler/src/test/resources/myriad-config-test-default-with-docker-info.yml
> <https://github.com/apache/incubator-myriad/pull/71#discussion_r65303137>:
>
> >                            # running the resource manager.
> >  frameworkSuperUser: root  # To be depricated, currently permissions need set by a superuser due to Mesos-1790.  Must be
> > -                          # root or have passwordless sudo. Required if nodeManagerURI set, ignored otherwise.
> > +                          # root or have passwordless sudo. Required if nodemanagerURI set, ignored otherwise.
>
> Looks like int MyriadExecutorConfiguration it's nodeManagerURI not
> nodemanagerURI, I like it better your way but we should be careful making
> configuration changes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/71/files/37b4df2dffec60b2825ed21754a56673eb195b03#r65303137>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKSRgbycUH1UPp1c011jiSXPHwFn4Rz-ks5qHRYigaJpZM4Ib_Am>
> .
>